addons/pos_loyalty/static/tests/tours/PosLoyaltyLoyaltyProgramTour.js

```diff
@@ -16,18 +16,18 @@ registry.category("web_tour.tours").add("PosLoyaltyLoyaltyProgram1", {
             // Order1: Generates 2 points.
             ProductScreen.addOrderline("Whiteboard Pen", "2"),
             ProductScreen.clickPartnerButton(),
-            ProductScreen.clickCustomer("Test Partner AAA"),
+            ProductScreen.clickCustomer("AAA Test Partner"),
```

Comment:
@odoo/pos Would it not be better to filter the partners another way?
Maybe using a `company_id` or something like that? (at least in the test)

I've already encountered the same issue in other PRs. Even if I found a solution then, it was annoying to debug and find the reason.

@jva-odoo Why do we suddenly need to sort? Where do the new partners come from?

Reply:
@william-andre I think it's from Indian demo data

Reply:
Maybe right now we do not need this fix but this can be improved
because now the Indian company is not loading by default